server/src/model/player.ts

```diff
@@ -7,6 +8,7 @@ export default class Player implements PlayerAttributes {
   public isBot: boolean;
   public isEliminated: boolean;
```

Comment:
Store the power ups in the rpsMatch class instead, you can do the same as what we did for actions.
![image](https://github.com/user-attachments/assets/b4b786e8-1122-4295-872c-653f097d8a73)

Just dont want to clutter up the player class with every variable we need for all the different games.